pfb-analysis/scripts/entrypoint.sh

```diff
@@ -38,6 +38,20 @@ do
     ((counter++))
 done
 
+PFB_TEMPDIR=`mktemp -d`
+
+# If given an S3 key for the shapefile, dowload and unzip it. Overrides PFB_SHPFILE.
+if [ "${PFB_SHPFILE_KEY}" ]
```

Comment:
It probably makes sense to make this more generic, such that we `wget` or `curl` a file provided to something like `PFB_SHPFILE_URL`. This way we could send in non-s3 download links. This also works with the current workflow because `job.boundary_file.url` returns a signed s3 download url valid for 24hrs, so we wouldn't need the aws CLI to retrieve this file.

I do think it makes sense that the download overwrites a locally provided file.